hotsos/core/plugins/storage/ceph.py

```diff
@@ -31,6 +32,16 @@
 
 CEPH_LOGS = "var/log/ceph/"
 
+CEPH_EOL_INFO = {
```

Comment:
If you could add a comment to say where you are getting these dates from that would be helpful for when we need to update them in the future.